west.yml

```diff
@@ -214,7 +214,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: af77d7cdfeeff290593e7e99f54f0c1e2a3f91e6
+      revision: pull/26/head
```

Comment:
PR was merged. Please use the latest hash.

Reply:
I updated it, thanks